actions/crib-deploy-environment/action.yml

```diff
@@ -148,28 +125,15 @@ runs:
         sanitized_branch=$(echo "${{ steps.generate-ns-name.outputs.branch-name }}" \
           | sed 's/[^a-zA-Z0-9._-]/_/g' | cut -c1-63 | sed 's/[._-]$//')
 
-        # Validate that chainlink-team and chainlink-product inputs adhere to the Kubernetes label format.
```

Comment:
You may need to rebase the branch.

Reply:
Rebased/pushed but I still see this comment 🤔

Reply:
Thanks! Fixed in 63d54bb824334f6f7537831c8d12a70905d8f28a.